### PR TITLE
Update database with santa assignments

### PR DESCRIPTION
**Pull request by JRodmanYoung**

### CODE REVIEW

File: db/schema.sql [hunk 1]
```diff
@@ -1,16 +1,18 @@
 CREATE TABLE people
 (
-	person_ID 		INTEGER PRIMARY KEY AUTOINCREMENT,
-	email			VARCHAR NOT NULL UNIQUE
+	person_ID 			INTEGER PRIMARY KEY AUTOINCREMENT,
+	email				VARCHAR NOT NULL UNIQUE,
+	givingTo			VARCHAR,
```
Comment (youngjl1) [inline db/schema.sql]: givingTo is the email? storing it is unnecessary. To retrieve the email you can always do something like:
`
select from.person_ID, from.email, from.givingTo_ID, to.email as givingToEmail
from people from
inner join people to
on from.givingTo_ID = to.person_ID
`

**Review by youngjl1 — CHANGES_REQUESTED** [review]
Looks good for the most part. You don't need to store givingToEmail as it's own db column however.

File: db/schema.sql (hunk 1 above)
Comment (JRodmanYoung) [reply db/schema.sql]: Ugh. I know, but I didn't want to. I'll update the database.

**Review by youngjl1 — APPROVED** [review]
Approved thanks. So the next step is to run a server which exposes your database and algorithm on a network. I want to talk more about what this means. Please find a time to call me.

**Pull request merged**